### PR TITLE
Sass Port

### PR DESCRIPTION
**Pull request by nicholas-b-carter**
As discussed this is the bootstrap-sass port of the library. Please let me know if you have any questions.

### CODE REVIEW

**Comment by scottmerker** [comment]
@nicholas-b-carter Nick,  I've created a new branch called gh-pages SASS port (experimental).  Would you mind doing a pull request for this into that one?  We want to make sure we don't effect any current ongoing projects by switching up the preprocessor on them.

Great work btw, looking forward to testing it out, at first pass I think this will be really valuable.

**Comment by nicholas-b-carter** [comment]
@scottmerker, sounds good. I'll change branches now and push it over. Thanks!

File: .idea/scssLintPlugin.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ScssLintProjectComponent">
+    <option name="scssLintConfigFile" value="$PROJECT_DIR$/_sass-lint.yml" />
+    <option name="pluginEnabled" value="true" />
+  </component>
+</project>
```
Comment (bmathews) [inline .idea/scssLintPlugin.xml]: Add to git ignore

File: .jshintrc [hunk 2]
```diff
@@ -17,5 +17,6 @@
     "strict": true,
     "trailing": true,
     "smarttabs": true,
-    "jquery": true
+    "jquery": true,
+    "asi": true
```
Comment (bmathews) [inline .jshintrc]: Could you remove this and add it in another PR/Issue if it's something you're concerned about?

Comment (nicholas-b-carter) [inline .jshintrc]: @bmathews, good catch! Yes, I will also add that to the ignore, sorry I thought it was already there. I am pushing this to the new branch per @scottmerker .

**Comment by scottmerker** [comment]
@nicholas-b-carter @bmathews Nick's port has been merged to the experimental branch. I'll close this pull request down.

**Pull request closed without merging**